io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl

```diff
@@ -0,0 +1,187 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
```

Comment:
for 2.0.0 version, new folder is missing. 
copyright headers are missing.
changes in release_notes required.